responsibleai/responsibleai/_managers/error_analysis_manager.py

```diff
@@ -202,9 +202,12 @@ def add(self, max_depth=3, num_leaves=31, filter_features=None):
         else:
             self._ea_config_list.append(ea_config)
 
+    @measure_time
     def compute(self):
         """Creates an ErrorReport by running the error analyzer on the model.
         """
+        print("Error Analysis")
```

Comment:
I don't think we should be adding prints?  maybe log.info or something, but not prints?

Reply:
The intension was to give users feedback as what computation is happening. Hence, added the prints.

The notebook output looks like this now:-

![image](https://user-images.githubusercontent.com/47334368/126361754-69999ea7-00f2-4445-b62d-9311476019ba.png)

Reply:
Looks really good. But I also wonder if maybe that can be off by default and have some way to turn it on from a notebook?

Reply:
I have it on by default. If we want to give user feedback we should leave it on by default.

Reply:
If a user calls this code in a script and they don't want to see the output can they turn it off? (Python SDKs usually avoid printing to stdout)

Reply:
That would be awesome. Maybe consider putting a `verbose`/`show_progress` flag in the constructor since we plan to remove the shared `compute()`?

Personally the way I'd do it is to implement a new `self._log()` method on `ModelAnalysis` that does the check for `self._verbose` to avoid repeating the `if self._verbose` check. But that's just an idea.

Reply:
closing this thread as deciding to just print the status.